Migration to Angular 13

konstantinschuette opened the pull request:
Hopefully this was not a waste of time, because the last PR (https://github.com/drozhzhin-n-e/ngx-pinch-zoom/pull/87) was ignored.

Things done:
- [X] Updated ALL packages & migration to Angular 13
- [X] Replaced `tslint` with `eslint`
- [X] Added Prettier / formatted all files with Prettier
- [X] Replaced Protractor with Cypress

Maybe someone should take care of more e2e tests. Because there are literally none

Commit:
updated all packages

Commit:
replaced tslint with eslint

Commit:
added prettier

Commit:
added .angular to gitignore and fixed unit test

Commit:
migrated protractor to cypress

Commit:
format all files with prettier

konstantinschuette commented:
If there is no respone after a week, I will cancel the PR and open up my own package.

turbolego commented:
> If there is no respone after a week, I will cancel the PR and open up my own package.

There is an Angular 13 package now, but it only goes up to Angular 13.2.x

https://www.npmjs.com/package/ngx-pinch-zoom-13

There's a need for a version of ngx-pinch-zoom that supports 13.3.x

`Package "ngx-pinch-zoom-13" has an incompatible peer dependency to "@angular/common" (requires "~13.2.1" (extended), would install "13.3.1").`

Raysix93 committed:
feat: Angular Update 13.3.x

konstantinschuette committed:
Merge pull request #1 from konstantinschuette/feature/angular_13.3.X_update

feat: Angular Update 13.3.x

Raysix93 committed:
feat: Added Information to Readme and package.json

Raysix93 committed:
republished with new version nr

konstantinschuette commented:
> > If there is no respone after a week, I will cancel the PR and open up my own package.
> 
> There is an Angular 13 package now, but it only goes up to Angular 13.2.x
> 
> https://www.npmjs.com/package/ngx-pinch-zoom-13
> 
> There's a need for a version of ngx-pinch-zoom that supports 13.3.x
> 
> `Package "ngx-pinch-zoom-13" has an incompatible peer dependency to "@angular/common" (requires "~13.2.1" (extended), would install "13.3.1").`

https://www.npmjs.com/package/@meddv/ngx-pinch-zoom

A bit late to the party, but here you go!

Commit:
updated dependencies to angular 14 and version bump

konstantinschuette committed:
Merge pull request #2 from TobiasJu/master

Updated dependencies to angular 14 and version bump

Commit:
Upgrade to latest Angular version

Raysix93 committed:
Merge pull request #4 from hoi4/angular14

Upgrade to latest Angular version

DwieDima committed:
fix: support all angular 14 versions

Raysix93 committed:
Merge pull request #6 from DwieDima/master

fix: support all angular 14 versions

Raysix93 committed:
doc: increment package version

dependabot[bot] committed:
Bump loader-utils from 2.0.2 to 2.0.3

Bumps [loader-utils](https://github.com/webpack/loader-utils) from 2.0.2 to 2.0.3.
- [Release notes](https://github.com/webpack/loader-utils/releases)
- [Changelog](https://github.com/webpack/loader-utils/blob/v2.0.3/CHANGELOG.md)
- [Commits](https://github.com/webpack/loader-utils/compare/v2.0.2...v2.0.3)

---
updated-dependencies:
- dependency-name: loader-utils
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #7 from medDV-GmbH/dependabot/npm_and_yarn/loader-utils-2.0.3

Bump loader-utils from 2.0.2 to 2.0.3

dependabot[bot] committed:
Bump loader-utils from 2.0.3 to 2.0.4

Bumps [loader-utils](https://github.com/webpack/loader-utils) from 2.0.3 to 2.0.4.
- [Release notes](https://github.com/webpack/loader-utils/releases)
- [Changelog](https://github.com/webpack/loader-utils/blob/v2.0.4/CHANGELOG.md)
- [Commits](https://github.com/webpack/loader-utils/compare/v2.0.3...v2.0.4)

---
updated-dependencies:
- dependency-name: loader-utils
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #8 from medDV-GmbH/dependabot/npm_and_yarn/loader-utils-2.0.4

Bump loader-utils from 2.0.3 to 2.0.4

dependabot[bot] committed:
Bump engine.io from 6.2.0 to 6.2.1

Bumps [engine.io](https://github.com/socketio/engine.io) from 6.2.0 to 6.2.1.
- [Release notes](https://github.com/socketio/engine.io/releases)
- [Changelog](https://github.com/socketio/engine.io/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/engine.io/compare/6.2.0...6.2.1)

---
updated-dependencies:
- dependency-name: engine.io
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #9 from medDV-GmbH/dependabot/npm_and_yarn/engine.io-6.2.1

Bump engine.io from 6.2.0 to 6.2.1

Commit:
package-lock update

Commit:
package-lock update

Commit:
ng update

Commit:
update @angular-eslint

Commit:
remove unnecessary enableIvy compiler option

Raysix93 committed:
Merge pull request #12 from hoi4/ng15

Ng15

Raysix93 committed:
fix: set correct readme

dependabot[bot] committed:
Bump json5 from 1.0.1 to 1.0.2

Bumps [json5](https://github.com/json5/json5) from 1.0.1 to 1.0.2.
- [Release notes](https://github.com/json5/json5/releases)
- [Changelog](https://github.com/json5/json5/blob/main/CHANGELOG.md)
- [Commits](https://github.com/json5/json5/compare/v1.0.1...v1.0.2)

---
updated-dependencies:
- dependency-name: json5
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #13 from medDV-GmbH/dependabot/npm_and_yarn/json5-1.0.2

Bump json5 from 1.0.1 to 1.0.2

dependabot[bot] committed:
Bump ua-parser-js from 0.7.32 to 0.7.33

Bumps [ua-parser-js](https://github.com/faisalman/ua-parser-js) from 0.7.32 to 0.7.33.
- [Release notes](https://github.com/faisalman/ua-parser-js/releases)
- [Changelog](https://github.com/faisalman/ua-parser-js/blob/master/changelog.md)
- [Commits](https://github.com/faisalman/ua-parser-js/compare/0.7.32...0.7.33)

---
updated-dependencies:
- dependency-name: ua-parser-js
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #14 from medDV-GmbH/dependabot/npm_and_yarn/ua-parser-js-0.7.33

Bump ua-parser-js from 0.7.32 to 0.7.33

dependabot[bot] committed:
Bump http-cache-semantics from 4.1.0 to 4.1.1

Bumps [http-cache-semantics](https://github.com/kornelski/http-cache-semantics) from 4.1.0 to 4.1.1.
- [Release notes](https://github.com/kornelski/http-cache-semantics/releases)
- [Commits](https://github.com/kornelski/http-cache-semantics/compare/v4.1.0...v4.1.1)

---
updated-dependencies:
- dependency-name: http-cache-semantics
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #15 from medDV-GmbH/dependabot/npm_and_yarn/http-cache-semantics-4.1.1

Bump http-cache-semantics from 4.1.0 to 4.1.1

dependabot[bot] committed:
Bump webpack and @angular-devkit/build-angular

Bumps [webpack](https://github.com/webpack/webpack) to 5.76.1 and updates ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Updates `webpack` from 5.75.0 to 5.76.1
- [Release notes](https://github.com/webpack/webpack/releases)
- [Commits](https://github.com/webpack/webpack/compare/v5.75.0...v5.76.1)

Updates `@angular-devkit/build-angular` from 15.0.3 to 15.2.8
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/15.0.3...15.2.8)

---
updated-dependencies:
- dependency-name: webpack
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

felix-ulonska committed:
npmi

felix-ulonska committed:
chore: updated to angular v16

Raysix93 committed:
Merge pull request #16 from medDV-GmbH/dependabot/npm_and_yarn/webpack-and-angular-devkit/build-angular-5.76.1

Bump webpack and @angular-devkit/build-angular

Raysix93 committed:
merge completed

Raysix93 committed:
fix: types/node error + updated readme

Raysix93 committed:
Updated readme

dependabot[bot] committed:
Bump socket.io-parser from 4.2.2 to 4.2.3

Bumps [socket.io-parser](https://github.com/socketio/socket.io-parser) from 4.2.2 to 4.2.3.
- [Release notes](https://github.com/socketio/socket.io-parser/releases)
- [Changelog](https://github.com/socketio/socket.io-parser/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io-parser/compare/4.2.2...4.2.3)

---
updated-dependencies:
- dependency-name: socket.io-parser
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #20 from medDV-GmbH/dependabot/npm_and_yarn/socket.io-parser-4.2.3

Bump socket.io-parser from 4.2.2 to 4.2.3

dependabot[bot] committed:
Bump word-wrap from 1.2.3 to 1.2.4

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.4.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.4)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #22 from medDV-GmbH/dependabot/npm_and_yarn/word-wrap-1.2.4

Bump word-wrap from 1.2.3 to 1.2.4

dependabot[bot] committed:
Bump vite and @angular-devkit/build-angular

Bumps [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite) to 4.3.9 and updates ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Updates `vite` from 4.3.1 to 4.3.9
- [Release notes](https://github.com/vitejs/vite/releases)
- [Changelog](https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md)
- [Commits](https://github.com/vitejs/vite/commits/v4.3.9/packages/vite)

Updates `@angular-devkit/build-angular` from 16.0.0 to 16.1.4
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/16.0.0...16.1.4)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #23 from medDV-GmbH/dependabot/npm_and_yarn/vite-and-angular-devkit/build-angular-4.3.9

Bump vite and @angular-devkit/build-angular

dependabot[bot] committed:
Bump @babel/traverse from 7.22.8 to 7.23.2

Bumps [@babel/traverse](https://github.com/babel/babel/tree/HEAD/packages/babel-traverse) from 7.22.8 to 7.23.2.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.23.2/packages/babel-traverse)

---
updated-dependencies:
- dependency-name: "@babel/traverse"
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #27 from medDV-GmbH/dependabot/npm_and_yarn/babel/traverse-7.23.2

Bump @babel/traverse from 7.22.8 to 7.23.2

dependabot[bot] committed:
Bump @cypress/request and cypress

Bumps [@cypress/request](https://github.com/cypress-io/request) to 3.0.1 and updates ancestor dependency [cypress](https://github.com/cypress-io/cypress). These dependencies need to be updated together.


Updates `@cypress/request` from 2.88.11 to 3.0.1
- [Release notes](https://github.com/cypress-io/request/releases)
- [Changelog](https://github.com/cypress-io/request/blob/master/CHANGELOG.md)
- [Commits](https://github.com/cypress-io/request/compare/v2.88.11...v3.0.1)

Updates `cypress` from 12.11.0 to 13.4.0
- [Release notes](https://github.com/cypress-io/cypress/releases)
- [Changelog](https://github.com/cypress-io/cypress/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/cypress-io/cypress/compare/v12.11.0...v13.4.0)

---
updated-dependencies:
- dependency-name: "@cypress/request"
  dependency-type: indirect
- dependency-name: cypress
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #28 from medDV-GmbH/dependabot/npm_and_yarn/cypress/request-and-cypress-3.0.1

Bump @cypress/request and cypress

dependabot[bot] committed:
Bump the npm_and_yarn at /. security update group with 1 update

Bumps the npm_and_yarn at /. security update group with 1 update: [@angular-devkit/build-angular](https://github.com/angular/angular-cli).

- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/16.2.9/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/16.1.4...16.2.9)

---
updated-dependencies:
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge pull request #29 from medDV-GmbH/dependabot/npm_and_yarn/angular-devkit/build-angular-16.2.9

Bump the npm_and_yarn at /. security update group with 1 update

Raysix93 committed:
feat: Angular 17 Support

Raysix93 committed:
feat: Angular 17

Vuki0901 committed:
Connect limitZoom with maxScale between PinchZoomComponent and IvyPinch. (#33)

* Connect maximum scale between component and IvyPinch.

* Replace if condition with nullish coalescing assignment in detectLimitZoom.

Raysix93 committed:
doc: updated version to 17.0.1

dependabot[bot] committed:
Bump vite and @angular-devkit/build-angular (#34)

Bumps [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite) to 4.5.1 and updates ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Updates `vite` from 4.5.0 to 4.5.1
- [Release notes](https://github.com/vitejs/vite/releases)
- [Changelog](https://github.com/vitejs/vite/blob/v4.5.1/packages/vite/CHANGELOG.md)
- [Commits](https://github.com/vitejs/vite/commits/v4.5.1/packages/vite)

Updates `@angular-devkit/build-angular` from 17.0.0 to 17.0.9
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/17.0.0...17.0.9)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump follow-redirects from 1.15.3 to 1.15.4

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.15.3 to 1.15.4.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.15.3...v1.15.4)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump vite and @angular-devkit/build-angular

Removes [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite). It's no longer used after updating ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Removes `vite`

Updates `@angular-devkit/build-angular` from 17.0.9 to 17.1.0
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/17.0.9...17.1.0)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump undici and @angular-devkit/build-angular

Bumps [undici](https://github.com/nodejs/undici) to 6.6.2 and updates ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Updates `undici` from 5.27.2 to 6.6.2
- [Release notes](https://github.com/nodejs/undici/releases)
- [Commits](https://github.com/nodejs/undici/compare/v5.27.2...v6.6.2)

Updates `@angular-devkit/build-angular` from 17.0.9 to 17.2.0
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/17.0.9...17.2.0)

---
updated-dependencies:
- dependency-name: undici
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump ip from 2.0.0 to 2.0.1

Bumps [ip](https://github.com/indutny/node-ip) from 2.0.0 to 2.0.1.
- [Commits](https://github.com/indutny/node-ip/compare/v2.0.0...v2.0.1)

---
updated-dependencies:
- dependency-name: ip
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/vite-and-angular-devkit/build-angular--removed'

Raysix93 committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/undici-and-angular-devkit/build-angular-6.6.2'

# Conflicts:
#	package-lock.json
#	package.json

Raysix93 committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/ip-2.0.1'

Raysix93 committed:
fix: package-lock

dependabot[bot] committed:
Bump follow-redirects from 1.15.3 to 1.15.6 (#40)

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.15.3 to 1.15.6.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.15.3...v1.15.6)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump webpack-dev-middleware from 5.3.3 to 5.3.4 (#42)

Bumps [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware) from 5.3.3 to 5.3.4.
- [Release notes](https://github.com/webpack/webpack-dev-middleware/releases)
- [Changelog](https://github.com/webpack/webpack-dev-middleware/blob/v5.3.4/CHANGELOG.md)
- [Commits](https://github.com/webpack/webpack-dev-middleware/compare/v5.3.3...v5.3.4)

---
updated-dependencies:
- dependency-name: webpack-dev-middleware
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump express from 4.18.2 to 4.19.2 (#43)

Bumps [express](https://github.com/expressjs/express) from 4.18.2 to 4.19.2.
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/master/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.18.2...4.19.2)

---
updated-dependencies:
- dependency-name: express
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump vite and @angular-devkit/build-angular (#44)

Removes [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite). It's no longer used after updating ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Removes `vite`

Updates `@angular-devkit/build-angular` from 17.2.0 to 17.3.3
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/17.2.0...17.3.3)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

gerhardboer committed:
feat: update to Angular 18 (#45)

* ng update @angular/cli

migrated to new build system

* update lib deps

* update versions of main package

Raysix93 committed:
doc: updated version in readme

dependabot[bot] committed:
Bump braces from 3.0.2 to 3.0.3 (#46)

Bumps [braces](https://github.com/micromatch/braces) from 3.0.2 to 3.0.3.
- [Changelog](https://github.com/micromatch/braces/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/braces/compare/3.0.2...3.0.3)

---
updated-dependencies:
- dependency-name: braces
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump ws, engine.io and socket.io-adapter (#47)

Bumps [ws](https://github.com/websockets/ws), [engine.io](https://github.com/socketio/engine.io) and [socket.io-adapter](https://github.com/socketio/socket.io-adapter). These dependencies needed to be updated together.

Updates `ws` from 8.17.0 to 8.17.1
- [Release notes](https://github.com/websockets/ws/releases)
- [Commits](https://github.com/websockets/ws/compare/8.17.0...8.17.1)

Updates `engine.io` from 6.5.4 to 6.5.5
- [Release notes](https://github.com/socketio/engine.io/releases)
- [Changelog](https://github.com/socketio/engine.io/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/engine.io/compare/6.5.4...6.5.5)

Updates `socket.io-adapter` from 2.5.2 to 2.5.5
- [Release notes](https://github.com/socketio/socket.io-adapter/releases)
- [Changelog](https://github.com/socketio/socket.io-adapter/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io-adapter/compare/2.5.2...2.5.5)

---
updated-dependencies:
- dependency-name: ws
  dependency-type: indirect
- dependency-name: engine.io
  dependency-type: indirect
- dependency-name: socket.io-adapter
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Commit:
type fixes: (#48)

- assigned proper types for all the 'any'
- changed let to const in all the appropriate places
- removed unused code and variables
- add 'private' access for all the internal stuff

dependabot[bot] committed:
Bump webpack and @angular-devkit/build-angular (#49)

Bumps [webpack](https://github.com/webpack/webpack) to 5.94.0 and updates ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Updates `webpack` from 5.91.0 to 5.94.0
- [Release notes](https://github.com/webpack/webpack/releases)
- [Commits](https://github.com/webpack/webpack/compare/v5.91.0...v5.94.0)

Updates `@angular-devkit/build-angular` from 18.0.2 to 18.2.2
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/18.0.2...18.2.2)

---
updated-dependencies:
- dependency-name: webpack
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump serve-static and express

Bumps [serve-static](https://github.com/expressjs/serve-static) and [express](https://github.com/expressjs/express). These dependencies needed to be updated together.

Updates `serve-static` from 1.15.0 to 1.16.2
- [Release notes](https://github.com/expressjs/serve-static/releases)
- [Changelog](https://github.com/expressjs/serve-static/blob/v1.16.2/HISTORY.md)
- [Commits](https://github.com/expressjs/serve-static/compare/v1.15.0...v1.16.2)

Updates `express` from 4.19.2 to 4.21.0
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/4.21.0/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.19.2...4.21.0)

---
updated-dependencies:
- dependency-name: serve-static
  dependency-type: indirect
- dependency-name: express
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump send and express

Bumps [send](https://github.com/pillarjs/send) and [express](https://github.com/expressjs/express). These dependencies needed to be updated together.

Updates `send` from 0.18.0 to 0.19.0
- [Release notes](https://github.com/pillarjs/send/releases)
- [Changelog](https://github.com/pillarjs/send/blob/master/HISTORY.md)
- [Commits](https://github.com/pillarjs/send/compare/0.18.0...0.19.0)

Updates `express` from 4.19.2 to 4.21.0
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/4.21.0/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.19.2...4.21.0)

---
updated-dependencies:
- dependency-name: send
  dependency-type: indirect
- dependency-name: express
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump vite and @angular-devkit/build-angular

Removes [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite). It's no longer used after updating ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Removes `vite`

Updates `@angular-devkit/build-angular` from 18.2.2 to 18.2.5
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/18.2.2...18.2.5)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump body-parser and express

Bumps [body-parser](https://github.com/expressjs/body-parser) and [express](https://github.com/expressjs/express). These dependencies needed to be updated together.

Updates `body-parser` from 1.20.2 to 1.20.3
- [Release notes](https://github.com/expressjs/body-parser/releases)
- [Changelog](https://github.com/expressjs/body-parser/blob/master/HISTORY.md)
- [Commits](https://github.com/expressjs/body-parser/compare/1.20.2...1.20.3)

Updates `express` from 4.19.2 to 4.21.0
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/4.21.0/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.19.2...4.21.0)

---
updated-dependencies:
- dependency-name: body-parser
  dependency-type: indirect
- dependency-name: express
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump cookie, socket.io and express

Bumps [cookie](https://github.com/jshttp/cookie), [socket.io](https://github.com/socketio/socket.io) and [express](https://github.com/expressjs/express). These dependencies needed to be updated together.

Updates `cookie` from 0.4.2 to 0.7.2
- [Release notes](https://github.com/jshttp/cookie/releases)
- [Commits](https://github.com/jshttp/cookie/compare/v0.4.2...v0.7.2)

Updates `socket.io` from 4.7.2 to 4.8.0
- [Release notes](https://github.com/socketio/socket.io/releases)
- [Changelog](https://github.com/socketio/socket.io/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io/compare/4.7.2...socket.io@4.8.0)

Updates `express` from 4.19.2 to 4.21.1
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/4.21.1/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.19.2...4.21.1)

---
updated-dependencies:
- dependency-name: cookie
  dependency-type: indirect
- dependency-name: socket.io
  dependency-type: indirect
- dependency-name: express
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Raysix93 committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/multi-9423f4c335'

Raysix93 committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/multi-cf87d80143'

Raysix93 committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/multi-d54fdff7c2'

# Conflicts:
#	package-lock.json

Raysix93 committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/multi-d66d039ac5'

# Conflicts:
#	package-lock.json

Raysix93 committed:
feat: Angular 19 Support, Switched to Standalone Component

dependabot[bot] committed:
build(deps): bump vite and @angular-devkit/build-angular (#56)

Removes [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite). It's no longer used after updating ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Removes `vite`

Updates `@angular-devkit/build-angular` from 19.0.4 to 19.0.4
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/19.0.4...19.0.4)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

mahnuh committed:
Implement zoom in and out methods (#60)

mahnuh committed:
Enable dragging while pinching (#59)

Raysix93 committed:
doc: #57 - documentation is now more accurate

Raysix93 committed:
v19.0.1

Raysix93 committed:
feat: #58 - added zoomChanged eventemitter as output

mahnuh committed:
Update package to support Angular 20 (#61)

* Set angular peer dependencies to 19 or higher

* Update readme

Raysix93 committed:
v20.0.0

dependabot[bot] committed:
build(deps): bump webpack-dev-server and @angular-devkit/build-angular (#62)

Bumps [webpack-dev-server](https://github.com/webpack/webpack-dev-server) to 5.2.1 and updates ancestor dependency [@angular-devkit/build-angular](https://github.com/angular/angular-cli). These dependencies need to be updated together.


Updates `webpack-dev-server` from 5.1.0 to 5.2.1
- [Release notes](https://github.com/webpack/webpack-dev-server/releases)
- [Changelog](https://github.com/webpack/webpack-dev-server/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack/webpack-dev-server/compare/v5.1.0...v5.2.1)

Updates `@angular-devkit/build-angular` from 19.0.4 to 20.0.1
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/19.0.4...20.0.1)

---
updated-dependencies:
- dependency-name: webpack-dev-server
  dependency-version: 5.2.1
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-version: 20.0.1
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Raysix93 committed:
fix package-json

dependabot[bot] committed:
build(deps): bump on-headers and compression (#64)

Bumps [on-headers](https://github.com/jshttp/on-headers) and [compression](https://github.com/expressjs/compression). These dependencies needed to be updated together.

Updates `on-headers` from 1.0.2 to 1.1.0
- [Release notes](https://github.com/jshttp/on-headers/releases)
- [Changelog](https://github.com/jshttp/on-headers/blob/master/HISTORY.md)
- [Commits](https://github.com/jshttp/on-headers/compare/v1.0.2...v1.1.0)

Updates `compression` from 1.7.5 to 1.8.1
- [Release notes](https://github.com/expressjs/compression/releases)
- [Changelog](https://github.com/expressjs/compression/blob/master/HISTORY.md)
- [Commits](https://github.com/expressjs/compression/compare/1.7.5...v1.8.1)

---
updated-dependencies:
- dependency-name: on-headers
  dependency-version: 1.1.0
  dependency-type: indirect
- dependency-name: compression
  dependency-version: 1.8.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump vite and @angular-devkit/build-angular (#65)

---
updated-dependencies:
- dependency-name: vite
  dependency-version: 
  dependency-type: indirect
- dependency-name: "@angular-devkit/build-angular"
  dependency-version: 19.0.4
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump form-data from 4.0.1 to 4.0.4 (#66)

---
updated-dependencies:
- dependency-name: form-data
  dependency-version: 4.0.4
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump js-yaml from 4.1.0 to 4.1.1 (#68)

Bumps [js-yaml](https://github.com/nodeca/js-yaml) from 4.1.0 to 4.1.1.
- [Changelog](https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md)
- [Commits](https://github.com/nodeca/js-yaml/compare/4.1.0...4.1.1)

---
updated-dependencies:
- dependency-name: js-yaml
  dependency-version: 4.1.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump tmp and @inquirer/editor (#70)

Bumps [tmp](https://github.com/raszi/node-tmp) and [@inquirer/editor](https://github.com/SBoudrias/Inquirer.js). These dependencies needed to be updated together.

Updates `tmp` from 0.2.3 to 0.2.5
- [Changelog](https://github.com/raszi/node-tmp/blob/master/CHANGELOG.md)
- [Commits](https://github.com/raszi/node-tmp/compare/v0.2.3...v0.2.5)

Updates `@inquirer/editor` from 4.2.0 to 4.2.23
- [Release notes](https://github.com/SBoudrias/Inquirer.js/releases)
- [Commits](https://github.com/SBoudrias/Inquirer.js/compare/@inquirer/editor@4.2.0...@inquirer/editor@4.2.23)

---
updated-dependencies:
- dependency-name: tmp
  dependency-version: 0.2.5
  dependency-type: indirect
- dependency-name: "@inquirer/editor"
  dependency-version: 4.2.23
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump node-forge from 1.3.1 to 1.3.2 (#71)

Bumps [node-forge](https://github.com/digitalbazaar/forge) from 1.3.1 to 1.3.2.
- [Changelog](https://github.com/digitalbazaar/forge/blob/main/CHANGELOG.md)
- [Commits](https://github.com/digitalbazaar/forge/compare/v1.3.1...v1.3.2)

---
updated-dependencies:
- dependency-name: node-forge
  dependency-version: 1.3.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Raysix93 committed:
feat - update package-lock

Raysix93 committed:
feat - angular update

Raysix93 committed:
feat: angular update

Raysix93 committed:
feat: updated peer dependencies for next version
